core/scheduler/testdata/TestSchedulerDuties_grouped.golden

```diff
@@ -2,8 +2,6 @@
  {
   "Duty": "0/proposer",
   "DutyArgSet": {
-   "0x8dae41352b69f2b3a1c0b05330c1bf65f03730c520273028864b11fcb94d8ce8f26d64f979a0ee3025467f45fd2241ea": "{\"pubkey\":\"0x8dae41352b69f2b3a1c0b05330c1bf65f03730c520273028864b11fcb94d8ce8f26d64f979a0ee3025467f45fd2241ea\",\"slot\":\"0\",\"validator_index\":\"2\"}",
```

Comment:
this is removed because this meant there are multiple block proposer for a single slot